Core/HLE/sceG729.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2012- PPSSPP Project.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 2.0 or later versions.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License 2.0 for more details.
+
+// A copy of the GPL 2.0 should have been included with the program.
+// If not, see http://www.gnu.org/licenses/
+
+// Official git repository and contact information can be found at
+// https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
+
+#include "Core/HLE/HLE.h"
+#include "Core/HLE/FunctionWrappers.h"
+
+const HLEFunction sceG729[] =
+{
+	{ 0x13F1028A, 0, "sceG729DecodeExit" },
+	{ 0x17C11696, 0, "sceG729DecodeInitResource" },
+	{ 0x3489D1F3, 0, "sceG729DecodeCore" },
+	{ 0x55E14F75, 0, "sceG729DecodeInit" },
```

Comment:
Again why do not work ?
I ensure there is a call "RegisterModule("sceG729", ARRAY_SIZE(sceG729), sceG729);" in HLETable.cpp
28:26:124 HLE\sceKernelModule.cpp:684 W[LOAD]: Unknown syscall in known module: sceG729 0x55e14f75